app/logger.js

```diff
@@ -1,64 +1,28 @@
-const { createLogger, format, transports } = require('winston');
-const { loggerLevel } = require('./config');
+import { createLogger, format, transports } from 'winston';
+import { loggerLevel } from './config';
 
-const logFormat = format.printf(({
-  level,
-  message,
-  label,
+const {
+  combine,
   timestamp,
-}) => `${timestamp} [${label}] ${level}: ${message}`);
+  label,
+  printf,
+  colorize,
+} = format;
+const logFormat = printf(info => `${info.timestamp} [${info.level}] ${info.label} | message: ${info.message} ${info.message.stack ? `: ${info.message.stack}` : ''}`);
 
 const logger = createLogger({
-  level: 'info',
-  format: logFormat,
-  defaultMeta: { service: 'user-service' },
-});
-
-if (process.env.NODE_ENV === 'development' || !process.env.NODE_ENV) {
-  logger.add(new transports.Console({
-    format: format.combine(
-      format.label({ label: 'marketing-pages' }),
-      format.timestamp({ format: 'D/M/YY HH:mm:ss' }),
-      format.colorize(),
-      logFormat,
-    ),
-  }));
-} else {
-  logger.add(
-    new transports.File({
-      filename: 'error.log',
-      level: 'error',
-      format: format.combine(
-        format.label({ label: 'marketing-pages' }),
-        format.timestamp({ format: 'D/M/YY HH:mm:ss' }),
-        logFormat,
-      ),
-    }),
-    new transports.File({
-      filename: 'combined.log',
-      format: format.combine(
-        format.label({ label: 'marketing-pages' }),
-        format.timestamp({ format: 'D/M/YY HH:mm:ss' }),
-        logFormat,
-      ),
+  format: combine(
+    label({ label: 'marketing-pages' }),
+    timestamp(),
+    colorize(),
+    logFormat,
+  ),
+  transports: [
+    new transports.Console({
+      level: loggerLevel,
+      colourize: (process.env.NODE_ENV === 'development' || !process.env.NODE_ENV),
```

Comment:
we could get this from the config?

Reply:
yeah

Reply:
😂 ok. Should we be using the config here then for env-vars rather than using them directly?

Reply:
that's what I meant by yeah 😄

Reply:
I waited for it be resolved and it never came. So thought you just meant yeah as in yeah